app/uk.gov.hmrc.helptosavestridefrontend/views/you_are_eligible.scala.html

```diff
@@ -48,13 +48,13 @@ <h1 class="heading-large">@messages("hts.you.are.eligible.h1")</h1>
             @{details.contactDetails.address1}<br />
             @{details.contactDetails.address2},<br />
             @if(details.contactDetails.address3.isDefined) {
```

Comment:
when dealing with `Option`s here does this not work?
```scala
@contactDeails.address3.map(_ <br />)
@contactDeails.address4.map(_ <br />)
```
Doing `if(option.isDefined){ ... }` is a bit ugly

Reply:
Thanks Andy, I have updated the syntax and pushed.